platform/darwin/src/MGLMapSnapshotter.mm

```diff
@@ -123,7 +150,13 @@ - (void)startWithQueue:(dispatch_queue_t)queue completionHandler:(MGLMapSnapshot
     __weak __typeof__(self) weakSelf = self;
     // mbgl::Scheduler::GetCurrent() scheduler means "run callback on current (ie UI/main) thread"
     // capture weakSelf to avoid retain cycle if callback is never called (ie snapshot cancelled)
-    _snapshotCallback = std::make_unique<mbgl::Actor<mbgl::MapSnapshotter::Callback>>(*mbgl::Scheduler::GetCurrent(), [=](std::exception_ptr mbglError, mbgl::PremultipliedImage image, mbgl::MapSnapshotter::Attributions attributions, mbgl::MapSnapshotter::PointForFn pointForFn) {
+
+    _snapshotCallback = std::make_unique<mbgl::Actor<mbgl::MapSnapshotter::Callback>>(*mbgl::Scheduler::GetCurrent(),
+                                                                                      [=](std::exception_ptr mbglError,
+                                                                                          mbgl::PremultipliedImage image,
+                                                                                          mbgl::MapSnapshotter::Attributions attributions,
+                                                                                          mbgl::MapSnapshotter::PointForFn pointForFn,
+                                                                                          mbgl::MapSnapshotter::LatLngForFn latLngForFn) {
```

Comment:
Nit: This indentation is a bit extreme. It’s technically correct (per Xcode), but I think it would be reasponable to dedent a bit, such as by indenting both parameters to `std::make_unique` by only two tabs.